test/parallel/test-crypto-lazy-transform-writable.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const crypto = require('crypto');
+const Stream = require('stream');
+const util = require('util');
+
+const hasher1 = crypto.createHash('sha256');
+const hasher2 = crypto.createHash('sha256');
+
+// calcualting the expected result
+hasher1.write(Buffer.from('hello world'));
+hasher1.end();
+
+const expected = hasher1.read().toString('hex');
+
+function OldStream() {
+  Stream.call(this);
+
+  this.readable = true;
+}
+util.inherits(OldStream, Stream);
+
+const stream = new OldStream();
+
+stream.pipe(hasher2).on('finish', common.mustCall(function() {
+  const hash = hasher2.read().toString('hex');
+  assert.strictEqual(expected, hash, 'the hash matches');
```

Comment:
Just my two bits, but I think most description strings are a waste of time and space. The reported on failure line number leads directly to the assert code, which is quite readable. As-is, the string just repeats the arg name and the word "match", which is a synonym for "equal", so its just a comment that restates exactly what the code says, a classic anti-pattern.